tests/org.eclipse.n4js.hlc.integrationtests/src/org/eclipse/n4js/hlc/integrationtests/CliJarWithVerdaccioTest.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2016 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.hlc.integrationtests;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.net.Socket;
+import java.nio.file.Path;
+
+import org.eclipse.n4js.N4JSGlobals;
+import org.eclipse.n4js.cli.helper.AbstractCliJarTest;
+import org.eclipse.n4js.cli.helper.ProcessResult;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * The tests in this class test n4js-cli npm.
+ */
+public class CliJarWithVerdaccioTest extends AbstractCliJarTest {
```

Comment:
would it make sense to already prepare an abstract class `AbstractCliJarWithVerdaccioTest` to separate that from the actual test cases in this class?